pkg/test/framework.go

```diff
@@ -102,7 +102,6 @@ func (opts *frameworkOpts) addToFlagSet(flagset *flag.FlagSet) {
 	flagset.StringVar(&opts.namespacedManPath, NamespacedManPathFlag, "", "path to rbac manifest")
 	flagset.BoolVar(&opts.isLocalOperator, LocalOperatorFlag, false,
 		"enable if operator is running locally (not in cluster)")
-	flagset.StringVar(&opts.kubeconfigPath, KubeConfigFlag, "", "path to kubeconfig")
```

Comment:
Missing changelog with the breaking change.

Reply:
Is it a breaking change? Sounds like `controller-runtime` is registring `--kubeconfig` so if a user passes in `--kubeconfig` the `controller-runtime` flag parser will set the value. 
And the change done to `main_entry.go` below will look to see if it is set and use the value that the user passed in that was captured by `controller-runtime`. 
If it isn't registered then we register it later.

Maybe I'm misunderstood how the flag parsing works but this change should be transparent to the user.